src/strings/PowerFxResources.en-US.resx

```diff
@@ -3621,6 +3621,17 @@
     <value>https://go.microsoft.com/fwlink/?linkid=2132570</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="ErrorResource_ErrBehaviorFunctionInDataUDF_ShortMessage" xml:space="preserve">
+    <value>Behavior function in a non-behavior user-defined function. Please wrap user-defined function with {} to declare a behavior UDF.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrorResource_ErrBehaviorFunctionInDataUDF_LongMessage" xml:space="preserve">
+    <value>Behavior functions change the state of the app by changing values elsewhere in the app. 'Navigate', 'Patch', 'UpdateContext', and 'Collect' are common behavior functions. 'OnSelect', 'OnVisible', and other 'On …' properties are common behavior-based properties.</value>
```

Comment:
>'Navigate', 'Patch', 'UpdateContext', and 'Collect' [](http://example.com/codeflow?start=99&length=51)

Similar to the previous comment, some of those functions are very Power Apps-specific. We should use examples of functions that are available in more hosts.